code/datums/commodity.dm

```diff
@@ -1243,6 +1243,15 @@
 	upperfluc = 50
 	lowerfluc = -50
 
+/datum/commodity/junk/pie_launcher
+	comname = "Pie Tool Module"
+	comtype = /obj/item/device/guardbot_tool/pie_launcher
+	desc = "A tool module compatible with guardbuddies. Are tool modules supposed to have cream on them?"
+	price = PAY_DOCTORATE/2
+	baseprice = PAY_DOCTORATE/2
+	upperfluc = PAY_DOCTORATE/6
+	lowerfluc = -PAY_DOCTORATE/6
```

Comment:
why base the price off doctorate instead of staffie or clown?

Reply:
because it's a Guardbuddy module, and RD messes with guardbuddies, not clown.